module/onboarding/Ping/src/Ping.cpp

```diff
@@ -25,7 +25,9 @@ namespace module::onboarding {
 
         on<Trigger<Pong>>().then([this](const Pong& pong_msg) {
             auto ping_msg = std::make_unique<Ping>();
-            log<NUClear::INFO>("Ping");
+            ping_msg->count = pong_msg.count + 1;
+            log<INFO>("Ping");
+            log<INFO>(ping_msg->count);
```

Comment:
`log` takes any number of arguments, so you can do 
```suggestion
            log<INFO>("Ping", ping_msg->count);
```
which might look neater in the output. It automatically puts spaces between each argument for you, so you don't need to manually add spaces.